tools/eslint-excludelist.json

```diff
@@ -31,7 +31,6 @@
 	"projects/plugins/jetpack/_inc/client/notices/index.js",
 	"projects/plugins/jetpack/_inc/client/pro-status/index.jsx",
 	"projects/plugins/jetpack/_inc/client/security/sso.jsx",
-	"projects/plugins/jetpack/_inc/client/state/at-a-glance/reducer.js",
```

Comment:
👍

Reply:
👍🏼

Reply:
@dognose24 I noticed this was preventing the build from passing, so the bot kept reapplying the 'Needs author reply' label. This seemed to sort it out!